integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/bundle/BundledClearAlarmRegister.feature

```diff
@@ -24,7 +23,6 @@ Feature: SmartMetering Bundle - ClearAlarmRegister
     And alarm register "1" of device "TEST1028000000002" has been cleared
     Examples:
       | protocol | version |
-      | DSMR     | 2.2     |
       | DSMR     | 4.2.2   |
```

Comment:
Now you should not remove 2.2 because you connect to 4.2.2 meter
| DSMR | 2.2 |

Reply:
This is corrected and DSMR 2.2 is added.